pkg/cmd/attestation/verification/sigstore_integration_test.go

```diff
@@ -43,31 +43,13 @@ func TestLiveSigstoreVerifier(t *testing.T) {
 	})
 
 	t.Run("with missing verification material", func(t *testing.T) {
-		attestations := getAttestationsFor(t, "../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-verification-material.jsonl")
-		require.NotNil(t, attestations)
-
-		verifier := NewLiveSigstoreVerifier(SigstoreConfig{
-			Logger: io.NewTestHandler(),
-		})
-
-		res := verifier.Verify(attestations, publicGoodPolicy(t))
-		require.Error(t, res.Error)
-		require.ErrorContains(t, res.Error, "failed to get bundle verification content")
-		require.Nil(t, res.VerifyResults)
+		_, err := GetLocalAttestations("../test/data/github_provenance_demo-0.0.12-py3-none-any-bundle-missing-verification-material.jsonl")
```

Comment:
@codysoyland @malancas should these files go into another test file? Glancing at this, since they don't `verify`, I don't think they hit the network, and therefore another file might be a better place for them than this one, that only runs when a special flag is provided to `go test`.

Reply:
Good point, I will move them in a follow-up PR!